README.md

```diff
@@ -273,6 +273,25 @@ List DKG status for each SKALE Chain on the node
 skale schains dkg
 ```
 
+#### SKALE Chain describe
+
+Show information about SKALE Chain on node
+```bash
+skale schains describe SCHAIN_NAME
```

Comment:
mb let's rename it to `skale schains info [SCHAIN_NAME]` or `skale schains inspect [SCHAIN_NAME]`?

Reply:
addressed